vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -90,8 +90,14 @@ testPackagesInstalled() {
     returnPackageVersions ${p} ${OS} ${OS_VERSION}
     PACKAGE_DOWNLOAD_URL=""
     returnPackageDownloadURL ${p} ${OS} ${OS_VERSION}
+    if [ ${name} == "kubernetes-binaries" ]; then
+      # kubernetes-binaries, namely, kubelet and kubectl are installed in a different way so we test them separately
+      testKubeBinariesPresent $PACKAGE_VERSIONS
+      echo "---"
```

Comment:
nit: whats the purpose of this echo? also, for any echo, can we do echo "$test <insert echo text>"
can look at other examples of echo in tests

Reply:
Ah right. This is not needed. Removed